src/cli.ts

```diff
@@ -95,13 +95,30 @@ function compile(args: any) {
         }
     }
 
-    if (args.disasm) {
-        const { isInstruction } = debugInfo!.info();
-        const disasm = disassemble(prg, { isInstruction });
-        for (const disasmLine of disasm) {
-            console.log(disasmLine);
+    if (args.disasm || args.disasmFile) {
+        let fd: number;
+        try{
+            console.log(`Generating ${args.disasmFile}`)
```

Comment:
This print should go under the below `if (args.disasmfile)` so that it's only printed when using `--disasm-file`.